core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java

```diff
@@ -95,6 +96,16 @@ long getCount() {
   }
 
   /**
+   * The sum of the squares of values in the population. If {@link #getCount()} is zero then this
+   * value will also be zero.
+   *
+   * @return The sum of squares of values in the population.
+   */
+  double getSumSquares() {
```

Comment:
Nit: `getSumOfSqurares()`?  `getSumSquares()` sounds a little ambiguous since it could also be the square of the sum of all values ((v1 + v2 + v3 ...) ^ 2).